include/motor-control/core/linear_motion_system.hpp

```diff
@@ -24,11 +24,13 @@ concept MotorMechanicalConfig = requires {
     std::is_same_v<MC, BeltConfig> || std::is_same_v<MC, LeadScrewConfig>;
 };
 
+// template <MotorEncoderConfig ECConfig>
 template <MotorMechanicalConfig MEConfig>
 struct LinearMotionSystemConfig {
     MEConfig mech_config{};
     float steps_per_rev{};
     float microstep{};
+    float encoder_ppr{};  // pulses_per_revolution
```

Comment:
if we're not confident that the name `encoder_ppr` gets it across, we should use a different name so we don't need the comment

Reply:
this is fine, just forgot to remove this comment.. PPR is standard for encoders, at least that's what I've found online.